src/components/App.js

```diff
@@ -4,17 +4,27 @@ import classNames from 'classnames';
 import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';
 import Fullscreen from 'react-full-screen';
 import { I18nextProvider } from 'react-i18next';
+import _ from 'lodash';
```

Comment:
Can you import lodash using a similar pattern here: https://github.com/ProjectMirador/mirador/blob/167b86007e48c8b207a48f9c74b7f810d12948c1/src/lib/ManifestoCanvas.js#L1

I think doing it that way limited the file size increases (in my testing at least a few weeks ago). I wonder if that is still the case.